src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Data/Migrations/20210902111722_Initial.cs

```diff
@@ -147,10 +154,10 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 name: "AspNetUserLogins",
                 columns: table => new
                 {
-                    LoginProvider = table.Column<string>(maxLength: 128, nullable: false),
-                    ProviderKey = table.Column<string>(maxLength: 128, nullable: false),
-                    ProviderDisplayName = table.Column<string>(nullable: true),
-                    UserId = table.Column<string>(nullable: false)
+                    LoginProvider = table.Column<string>(type: "TEXT", maxLength: 128, nullable: false),
+                    ProviderKey = table.Column<string>(type: "TEXT", maxLength: 128, nullable: false),
+                    ProviderDisplayName = table.Column<string>(type: "TEXT", nullable: true),
+                    UserId = table.Column<string>(type: "TEXT", nullable: false)
```

Comment:
These sort of changes seem unrelated to the PR title. For our longer-term tracking, could you rename the PR (or at least the squashed commit message) to something more accurate?

Reply:
I had to fix this as part of my testing. Migrations EF weren't agreeing with me